engine/graphics/src/null/graphics_null.cpp

```diff
@@ -1043,7 +1041,11 @@ namespace dmGraphics
                 uint32_t buffer_size                    = GetBufferSize(params.m_ColorBufferParams[i]);
                 rt->m_ColorTextureParams[i].m_DataSize  = buffer_size;
                 rt->m_ColorBufferTexture[i]             = NewTexture(context, params.m_ColorBufferCreationParams[i]);
-                Texture* attachment_tex                 = GetAssetFromContainer<Texture>(context->m_AssetHandleContainer, rt->m_ColorBufferTexture[i]);
+                Texture* attachment_tex = 0x0;
+                {
+                    DM_MUTEX_SCOPED_LOCK(g_NullContext->m_AssetContainerMutex);
+                    attachment_tex                 = GetAssetFromContainer<Texture>(context->m_AssetHandleContainer, rt->m_ColorBufferTexture[i]);
```

Comment:
I'm starting to think we should have a container or helper function that supports this workflow.